code/modules/vending/autodrobe.dm

```diff
@@ -1,214 +1,27 @@
 /obj/machinery/vending/autodrobe
-	name = "\improper AutoDrobe"
-	desc = "A vending machine for costumes."
+	name = "\improper Paperdrobe"
+	desc = "A vending machine for removing fun."
 	icon_state = "theater"
 	icon_deny = "theater-deny"
 	req_access = list(ACCESS_THEATRE)
-	product_slogans = "Dress for success!;Suited and booted!;It's show time!;Why leave style up to fate? Use AutoDrobe!"
+	product_slogans = "Get back to work, slave!;Paperwork is love. Paperwork is life.;I love roleplay.;THIS IS GOOD FOR MRP BECAUSE"
```

Comment:
```suggestion
	product_slogans = "Get back to work, slave!;Paperwork is love. Paperwork is life.;I love roleplay.;THIS IS GOOD FOR MRP BECAUSE"
```
Please change this desc, it's very lrp and meta